test/itkFrequencyBandPassImageFilterTest.cxx

```diff
@@ -72,12 +72,10 @@ int itkFrequencyBandPassImageFilterTest( int, char*[] )
   TEST_SET_GET_VALUE( passBand, bandPassFilter->GetPassHighFrequencyThreshold() );
 
   bool passLowFreqThreshold = true;
-  bandPassFilter->SetPassLowFrequencyThreshold( passLowFreqThreshold );
-  TEST_SET_GET_VALUE( passLowFreqThreshold, bandPassFilter->GetPassLowFrequencyThreshold() );
+  TEST_SET_GET_BOOLEAN( bandPassFilter, PassLowFrequencyThreshold, passLowFreqThreshold );
```

Comment:
Ups, sorry, I was building locally with my branch forked from an old master. There was no `TEST_SET_GET_BOOLEAN` in `itkTestingMacro.h` at that time.